README.rst

```diff
@@ -74,11 +74,27 @@ The easiest way to install Nansat on a Linux machine is to use
     # update miniconda packages and metadata
     conda update -q --yes conda
 
-    # install all requirements from conda-forge channel
-    conda install -q --yes -c conda-forge gdal numpy pillow netcdf4 python-dateutil pythesint nose
+    # install Nansat and requirements from the conda-forge channel
+    conda create -n py3nansat -c conda-forge nansat
+    
+    # activate the py3nansat environment
+    source activate py3nansat    
 
-    # finally install Nansat
-    pip install https://github.com/nansencenter/nansat/archive/master.tar.gz
+Fore more information see
+`Install-Nansat <https://github.com/nansencenter/nansat/wiki/Install-Nansat>`__
+section or use pre-configure virtual machines as explained on
+`Nansat-lectures <https://github.com/nansencenter/nansat-lectures>`__
+
```

Comment:
These four lines can be removed.
The link to wikipage is broken. The Nansat-lectures provide wrong docs - outdated Vagrant configuration.

Reply:
done